types/mui-datatables/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "private": true,
+    "dependencies": {
+        "@material-ui/core": ">=3",
+        "csstype": "^2.0.0",
+        "indefinite-observable": "^1.0.1"
```

Comment:
Why are csstype and Indefinite-Observable a dependency now? I can sort of understand Material-ui/core due to your typings but I don't see these in your changes at all.

Reply:
I might actually need some help here. The two dependencies should be removed. I only added them trying to troubleshoot the error I'm getting when I run `npm run tslint mui-datatables`:

```
Error: Errors in typescript@next for external dependencies:
../jss/css.d.ts(3,28): error TS2307: Cannot find module 'indefinite-observable'.
../jss/css.d.ts(4,26): error TS2307: Cannot find module 'csstype'.

    at /Users/petrbela/code/petrbela/DefinitelyTyped/node_modules/dtslint/bin/index.js:189:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/petrbela/code/petrbela/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
```

Same for `npm run test`. (Not sure why the CI didn't run here?)

`@material-ui/core` should pull in `csstype` and `indefinite-observable` as transitive dependencies via the `jss` package but it's not happening for some reason. Do you know why that might be?